core/templates/dev/head/css/oppia.css

```diff
@@ -3901,6 +4214,46 @@ md-card.preview-conversation-skin-supplemental-card {
   right: 0;
 }
 
+/*
+  Styles for Setting tab
+*/
+.oppia-settings-container .has-error .form-text {
+  color: #a94442;
+}
+
+.oppia-settings-container .form-text {
+  color: #737373;
+  display: inline-block;
+  margin-top: 5px;
+  margin-bottom: 15px;
+}
+.has-error .form-control {
```

Comment:
These seem to apply more generally than the Settings tab (there is no "settings" qualifier on the CSS rule selector).

Reply:
Moved to the correct place